packages/mc-html-template/src/process-headers.js

```diff
@@ -120,7 +120,9 @@ const processHeaders = (applicationConfig) => {
   );
 
   return {
-    'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload',
```

Comment:
So the `preload` option is now NOT a default anymore? Any particular reason for that?

Reply:
The preload wasn't default before. The steps we took where while we only had `max-age`:

1. Add includeSubDomains and preload
2. Reflect and consider making it configurable and hence remove the just added defaults

All cause we wanted to integrate this change slowly one app first. Now that it runs for a week or two we could remove the configuration option or change the default.

Reply:
Ah, I didn't see the other PR #2242 🤦‍♂️